tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -28,7 +28,44 @@ use tracing_core::{
 
 /// A `Layer` which filters spans and events based on a set of filter
 /// directives.
-// TODO(eliza): document filter directive syntax?
+///
+/// # Directives
+///
+/// A filter consists of one or more directives. Directives match [`Span`]s and [`Event`]s
+/// and specify a maximum verbosity [`level`] to enable for those that match. The directive
```

Comment:
since this is lowercase, i would probably _not_ put backticks around it here
```suggestion
/// and specify a maximum verbosity [level] to enable for those that match. The directive
```